### PR TITLE
REL: 1.1.6

### PR DESCRIPTION
**Pull request by effigies**
## Summary

Prep for 1.1.6 release, targeting Monday, November 26.

## Release checklist

* [x] Update changelog
* [x] Update .mailmap
* [x] Update .zenodo.json
* [x] Set release number in `nipype/info.py` and `doc/conf.py`
* [x] Update `doc/documentation.rst` with previous releases
* [x] Tutorial tests (@djarecka)

## Acknowledgment

- [x] \(Mandatory\) I acknowledge that this contribution will be available under the Apache 2 license.

### CODE REVIEW

File: nipype/info.py [hunk 1]
```diff
@@ -136,6 +137,7 @@ def get_nipype_gitversion():
     'nibabel>=%s' % NIBABEL_MIN_VERSION,
     'networkx>=%s' % NETWORKX_MIN_VERSION,
     'numpy>=%s' % NUMPY_MIN_VERSION,
+    'numpy>=%s ; python_version >= "3.7"' % NUMPY_MIN_VERSION_37,
```
Comment (effigies) [inline nipype/info.py]: @satra @oesteban This is intended to address https://github.com/nipy/nipype/pull/2775#issuecomment-438312643. I've tested locally with pip installs of the `bdist` wheel in different conda environments. Added a test to CI, so we'll see how that goes.

File: .zenodo.json [hunk 2]
```diff
@@ -451,12 +456,6 @@
       "name": "Durnez, Joke",
       "orcid": "0000-0001-9030-2202"
     },
-    {
-      "name": "Mertz, Fred"
-    },
```
Comment (effigies) [inline .zenodo.json]: From earlier comment, this appears to be a psuedonym. Let me know if I should restore it.

**Comment by effigies** [comment]
@djarecka This is in final state, as far as I'm concerned. If you haven't already, could you start the tutorial tests?

**Comment by djarecka** [comment]
@effigies - I checked the version from 2 hours ago, don't think you've changed anything since then that can affect the tutorial

**Comment by effigies** [comment]
@djarecka Sounds good. Thanks.

@nipy/team-nipype Will merge and release on passing tests unless anyone objects.

**Pull request merged**